config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+mclb = Money::Bank::CurrencylayerBank.new
+mclb.access_key = ENV.fetch("CURRENCY_LAYER_API_KEY", "")
```

Comment:
This should be `AppSecrets.CURRENCYLAYER_API_KEY` instead. We have a top-level file called `app_secrets.rb` where you can see how we configure secrets access.

In dev and test, the values for these secrets are loaded from environment variables, which means placing it in `.env.development` and `.env.test` is correct. For production (and staging), you tell me the real access key through a (semi-)secure channel, and I insert it into a Vault runtime that feeds our production servers